scripts/page_browse.js

```diff
@@ -384,11 +378,10 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false) {
                     // by the shown (first) option
                     const round = roundSelector.value;
                     params.set("round_id", round);
-                    $.ajax(makeApiAjaxSettings("v1/projects/" + projectId + "/pages/" + imageFileName + "/pagerounds/" + round))
-                        .done(function(data) {
+                    ajax("GET", `v1/projects/${projectId}/pages/${imageFileName}/pagerounds/${round}`)
```

Comment:
does the image file name ever need encoding to handle characters not valid in url path params like '/' or something that may need encoding?

Reply:
Currently our image names only contain numbers/letters + period + ext. I'm comfortable with using it as-is in the URL.